.travis.yml

```diff
@@ -1,10 +1,35 @@
 language: rust
-
-rust:
-  - stable
-  - beta
-  - nightly
+services: docker
+sudo: required
 
 matrix:
-  allow_failures:
-    - rust: nightly
+  include:
+    - env: TARGET=i686-unknown-linux-gnu
+      rust: stable
+    - env: TARGET=x86_64-unknown-linux-gnu
+      rust: stable
+    - env: TARGET=x86_64-unknown-linux-gnu
+      rust: 1.21.0
+    - env: TARGET=x86_64-unknown-linux-gnu
+      rust: nightly
+    - env: TARGET=powerpc-unknown-linux-gnu
+      rust: stable
+    - env: TARGET=powerpc64-unknown-linux-gnu
+      rust: stable
+    # tests if crates truly can be built without std
+    - env: TARGET=thumbv7em-none-eabi
```

Comment:
This test should be removed, as currently `hkdf` can not be used in no_std environments.